project/EnsoProjects.scala

```diff
@@ -0,0 +1,64 @@
+import sbt.*
+import sbt.nio.file.FileTreeView
+
+import java.nio.file.Path
+
+/** Helper for locating Enso projects stored in the source tree.
+  *
+  * The projects include standard libraries, test suites, benchmarks.
+  */
+object EnsoProjects {
+  case class Project(namespace: Option[String], name: String, path: Path) {
+    def usesPrivateAccess: Boolean =
+      path.getParent.getFileName.toString == "test" && name.contains(
```

Comment:
`.toString` obsolete?

Reply:
Feels like a regex may be more appropriate here

Reply:
> `.toString` obsolete?

![image](https://github.com/user-attachments/assets/559b57d7-ff5d-41ea-ac16-81138aefbcb6)

> Feels like a regex may be more appropriate here
I've used the same logic that we had elsewhere. I will try to make it a common method. IMO regex is an overkill for such a simple check - the two checks are more readable than something like `_Internal_(.*_)+Tests`